templates/web/pages/bmviii_home.tmpl

```diff
@@ -0,0 +1,342 @@
+{{ define "bmviii home" }}
+{{ template "head" . }}
+
+{{ template "bmviii navbar" . }}
+
+{{ template "flashes" .Flashes }}
+{{ template "mlh badge" }}
```

Comment:
This is still the 2020 MLH badge, I think we'll be part of the 2021 season

Reply:
Do you know where to get an image of the 2021 badge?